sirbot-pyslackers/endpoints/slack/messages.py

```diff
@@ -53,9 +53,9 @@ def create_endpoints(plugin):
             LOG.error("Error retrieving stock quotes: %s", e)
             response["text"] = "Unable to retrieve quotes right now."
     else:
-        change = quote["change"]
+        change = quote.get("change", 0)
```

Comment:
`quote["change"]` is used line `74` as well and `quote["changePercent"]` might also be `None` I guess.

I don't know what a change of `None` means, no change or ?
Anyways it might be better to skip displaying the `Change` field if the change is `None` ?

If you feel like it putting a `LOG.debug` in the stock plugin could be useful to have an idea of the exact payload.